### ISSUE TITLE
Eroor when trying to use biblioNetwork(M, analysis = "coupling", network = "references", sep = ";")

### ISSUE THREAD

**Issue by Margu006** [massimoaria/bibliometrix#137]
Hi bibliometrix creators, 

I want to create a coupling network based on whether the papers on my database share references among them. For this, I have tried to do a coupling of references running this code:

biblioNetwork(M, analysis = "coupling", network = "references", sep = ";")

However, the outcome is the following error:
Error in dimnamesGets(x, value) : 
  invalid dimnames given for “dgCMatrix” object

I am working with bib files that have been downloaded from Web of Science. I get this error only with this specifications for analysis and network. I am able to produce coupling of authors (biblioNetwork(M, analysis = "coupling", network = "authors", sep = ";")) and co-citation networks. 

Any suggestion to fix this?

**Comment by massimoaria** [comment]
Please attach your code and bib file.

**Comment by Margu006** [comment]
Hi Massimo, 

Thank you for your response. I found the cause of the error in the data during the weekend. I had merged the bib files using mergeDBSources function which outcome is a data.frame but not a bibliometrixDB. Then, when trying to use biblionetwork function, the package couldn't find the class required.

Thanks anyways, I appreciate a lot your availability and the time taken to answer!
I'll close the issue
